config/qualifications.yml

```diff
@@ -17,15 +16,13 @@ description: >
 subtasks:
   - '[Security Checklist](https://www.umass.edu/it/support/security/security-checklist-university-owned-computers)'
   - '[Understanding Sensitive Data](https://www.umass.edu/it/support/security/understand-sensitive-data-umass-amherst)'
-  - '[Data Classification]9https://www.umass.edu/it/support/security/data-classification-umass-amherst)'
+  - '[Data Classification](https://www.umass.edu/it/support/security/data-classification-umass-amherst)'
 ---
 title: 'Read the transportation programmer policies'
 description: >
-  Ask your supervisor to send you this document via email if they have not
-  already done so. It contains policies regarding attendance, remote work, etc.
+  This document lives in our [box](https://umass.app.box.com) folder called 'Programmer Training Documents'.
```

Comment:
BTW, you can link to a specific folder or file in Box without making the file "public".